ena-submission/scripts/submission_db_helper.py

```diff
@@ -139,12 +156,16 @@ def find_errors_in_db(db_conn_pool, table_name, time_threshold=15):
     try:
         with con, con.cursor(cursor_factory=RealDictCursor) as cur:
             min_start_time = datetime.now(tz=pytz.utc) + timedelta(minutes=-time_threshold)
+            # Prevent sql-injection with table_name validation
+            TableName.validate(table_name)
 
-            query = f"SELECT * FROM {table_name} WHERE "
-            query += f"(status='HAS_ERRORS' AND started_at < timestamp '{min_start_time}')"
-            query += f" OR (status='SUBMITTING' AND started_at < timestamp '{min_start_time}')"
+            query = f"""
+                SELECT * FROM {table_name} 
+                WHERE (status = %s AND started_at < %s)
+                OR (status = %s AND started_at < %s)
+            """
 
-            cur.execute(query)
+            cur.execute(query, ("HAS_ERRORS", min_start_time, "SUBMITTING", min_start_time))
```

Comment:
To improve readability, the constants such as `"HAS_ERRORS"` can be moved directly into the SQL. Otherwise, this looks great and safe!